### ISSUE TITLE
Color contrast bug in modals

### ISSUE THREAD

**Issue by arjunkalburgi** [avanslaars/cypress-axe#52]
I have a modal that overlays the screen: 
<img width="274" alt="image" src="https://user-images.githubusercontent.com/8989414/89322885-ecfeb800-d652-11ea-9d55-f8f4e0eeed4f.png">

(note backdrop darkens) 

I'm testing for a11y like so: `cy.checkA11y(".modal", null, terminalLog);`

But always get this contrast error: 

```
failureSummary: "Fix any of the following:
  Element has insufficient color contrast of 2.92 (foreground color: #979797, background color: #ffffff, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1"
```

The font size is 18px, the text is #000 and the background is #FFF. I'm not sure how it's getting this #979797 value. This bug only happens when I am checking my modal element for a11y, so I think something is going wrong when I pass in the element I want to check.

**Comment by sapegin** [comment]
Are you sure this is a problem with cypress-axe specifically and not axe-core itself that is doing the check? Sometimes Axe cannot correctly determine the background color.

**Comment by stereobooster** [comment]
https://github.com/dequelabs/axe-core/issues?q=is%3Aissue+is%3Aopen+contrast+label%3A%22color+contrast%22+modal ?
